src/app/common/components/PipelineSummary.css

```diff
@@ -0,0 +1,12 @@
+.dash {
+  height: 2px;
+  width: 10px;
+}
+
+.dashReached {
+  background-color: rgb(56, 139, 56);
```

Comment:
Actually we should probably use PatternFly CSS variables for these colors so we don't have to hard-code the matching rgb values: https://www.patternfly.org/v4/guidelines/colors

I think maybe for `dashReached` it would be `background-color: var(--pf-global--success-color--100)` and for `dashNotReached` it would be `background-color: var(--pf-global--disabled-color--200)`

Reply:
I was wondering about that too. Great I'll use PF colors. Thanks

Reply:
That way I don't have to do an exact color pick! ;)